src/blockchain_processor.py

```diff
@@ -553,6 +553,13 @@ def process(self, request, cache_only=False):
             num = int(params[0])
             result = self.dashd('estimatefee', [num])
 
+        # Masternode methods.
+
+        elif method == 'masternode.announce.broadcast':
+            # masternodebroadcast expects a vector.
+            mn = '01' + str(params[0])
```

Comment:
I think it might be better to make this message more generic and move `'01' + ` part to the client so that we could use the same message for cases where it carries multiple broadcasts at once.